build/ferretdb/production.Dockerfile

```diff
@@ -88,6 +88,7 @@ go version -m bin/ferretdb
 bin/ferretdb --version
 EOF
 
+RUN mkdir /state
```

Comment:
Please move it to the block above to avoid creating another layer

Reply:
fixed